src/pointer.rs

```diff
@@ -69,12 +79,12 @@ impl PenInclination {
     // Reference for the conversion functions:
     // https://www.w3.org/TR/pointerevents3/#converting-between-tiltx-tilty-and-altitudeangle-azimuthangle
 
-    pub fn from_tilt(tilt_x: i8, tilt_y: i8) -> Option<PenInclination> {
+    pub fn from_tilt(tilt_x: f64, tilt_y: f64) -> Option<PenInclination> {
         use std::f64::consts::{PI, TAU};
-        let tilt_x_rad = tilt_x as f64 * PI / 180.0;
-        let tilt_y_rad = tilt_y as f64 * PI / 180.0;
+        let tilt_x_rad = tilt_x * PI / 180.0;
+        let tilt_y_rad = tilt_y * PI / 180.0;
 
-        if tilt_x.abs() == 90 || tilt_y.abs() == 90 {
+        if tilt_x.abs() > 89.0 || tilt_y.abs() > 89.0 {
```

Comment:
I don't see `89` in the reference implementation linked above — maybe worth adding a comment indicating why we're not using 90, and how 89 specifically was chosen?